install/tests/default-install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Tests that an unconfigured installation works properly.
+# WARNING: this test makes changes to the local filesystem and is intended to be run in a containerized environment.
+
+set -o errexit
+source ./test-util.sh
+
+cleanup () {
+    if has_command "swiftly" ; then
+       swiftly uninstall -y latest > /dev/null
+    fi
+
+    rm -r "$HOME/.local/share/swiftly"
+    rm -r "$HOME/.local/bin/swiftly"
+}
+trap cleanup EXIT
+
+export PATH="$HOME/.local/bin:$PATH"
+
+echo "1" | ./swiftly-install.sh
+
+if ! has_command "swiftly" ; then
+    test_fail "Can't find swiftly on the PATH"
+fi
+
+if [[ ! -d "$HOME/.local/share/swiftly/toolchains" ]]; then
+    test_fail "the toolchains directory was not created in SWIFTLY_HOME_DIR"
+fi
+
+swiftly install latest
+
+swift --version
+
+test_pass
```

Comment:
The `test_pass` function used to print some stuff that I ended up moving to `run-tests.sh`, so now it doesn't really do anything. I figured I'd leave it in if we ever wanted to do some passed-test specific stuff though.